src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -1,231 +1,852 @@
-      return <div className="security-tab-empty">
-        <p className="security-tab-empty__title">Scan is Disabled</p>
-      </div>
-    }
-    else if (mat.vulnerabilitiesLoading) {
-      return <div className="security-tab-empty">
-        <Progressing />
-      </div>
-    }
-    else if (!mat.vulnerabilitiesLoading && mat.vulnerabilities.length === 0) {
-      return <div className="security-tab-empty">
-        <p className="security-tab-empty__title">No vulnerabilities Found</p>
-        <p className="security-tab-empty__subtitle">{mat.lastExecution}</p>
-      </div>
-    }
-    else return <div className="security-tab">
-      <p className="security-tab__last-scanned">Scanned on {mat.lastExecution} </p>
-      <ScanVulnerabilitiesTable vulnerabilities={mat.vulnerabilities} />
-    </div>
-  }
-
-  renderSequentialCDCardTitle = (mat) => {
-    if (this.props.stageType !== 'CD') return;
-
-    if (mat.latest && mat.runningOnParentCd) {
+    renderMaterialInfo(mat: CDMaterialType, hideSelector?: boolean) {
         return (
-            <div className="bcv-1 pt-6 pb-6 pl-16 pr-16 br-4">
-                <span className="cn-9 fw-6">Deployed on </span>{' '}
-                <span className="cv-5 fw-6">
-                    {this.props.parentEnvironmentName}
-                    {this.props.parentEnvironmentName ? (
+            <>
+                <div className="flex left column">
+                    <div className="commit-hash commit-hash--docker">
+                        <img src={docker} alt="" className="commit-hash__icon" />
+                        {mat.image}
+                    </div>
+                    {this.props.stageType !== 'CD' && mat.latest && (
+                        <span className="last-deployed-status">Last Run</span>
+                    )}
+                </div>
+                {this.props.materialType === 'none' ? (
+                    <div />
+                ) : (
+                    <div className="material-history__info flex left fs-13">
+                        <DeployIcon className="icon-dim-16 scn-6 mr-8" />
+                        <span className="fs-13 fw-4">{mat.deployedTime}</span>
+                    </div>
+                )}
+                {!!mat.deployedBy && this.state.isRollbackTrigger ? (
+                    <div className="material-history__deployed-by flex left">
+                        {mat.deployedBy === 'system' ? (
+                            <>
+                                <BotIcon className="icon-dim-16 mr-6" />
+                                <span className="fs-13 fw-4">Auto triggered</span>
+                            </>
+                        ) : (
+                            <>
+                                <span
+                                    className="flex fs-13 fw-6 lh-18 icon-dim-20 mr-6 cn-0 m-auto"
+                                    style={{
+                                        backgroundColor: getRandomColor(mat.deployedBy),
```

Comment:
Inline CSS please remove

Reply:
Removed inline CSS except `backgroundColor: getRandomColor(mat.deployedBy)` as this will be dynamic.